src/blocks/content-helper/content-helper-error.tsx

```diff
@@ -52,14 +66,58 @@ export class ContentHelperError extends Error {
 		Object.setPrototypeOf( this, ContentHelperError.prototype );
 	}
 
-	public ProcessedMessage( className = '' ): JSX.Element {
-		// Errors that need to display the "Contact Us" message.
-		const contactUsErrorCodes = [
+	/**
+	 * Returns an error message element that can contain HTML.
+	 *
+	 * Warning: Any HTML passed to this function should be sanitized.
+	 *
+	 * @since 3.9.0
+	 *
+	 * @param {ContentHelperErrorMessageProps} props The props needed for the component.
+	 *
+	 * @return {JSX.Element} The resulting JSX Element.
+	 */
+	protected ContentHelperErrorMessage( props: ContentHelperErrorMessageProps ): JSX.Element {
+		let innerHtml = '';
+		if ( props.children ) {
+			innerHtml = props.children;
+		}
+
+		let classNames = 'content-helper-error-message';
+		if ( props.className ) {
+			classNames += ' ' + props.className;
+		}
+
+		return (
+			<div className={ classNames }
+				data-testid={ props?.testId }
+				dangerouslySetInnerHTML={ { __html: innerHtml } }
+			/>
+		);
+	}
+
+	/**
+	 * Renders the error's message.
+	 *
+	 * @param {ContentHelperErrorMessageProps|null} props The props needed for the function.
+	 *
+	 * @return {JSX.Element} The resulting JSX Element.
+	 */
+	public renderMessage( props: ContentHelperErrorMessageProps|null = null ): JSX.Element {
+		// Handle cases where credentials are not set.
+		const CredentialsNotSetErrorCodes = [
```

Comment:
We're currently using this like a function rather than a component, thus the camelCase and the `render` in the name. The `error` object (from which this class extends) has a `message` property itself, so this name was picked to distinguish it from that.

Maybe there's a better way to restructure everything and give better names, but I'm unsure it would be part of this PR.

Reply:
Please see https://github.com/Parsely/wp-parsely/pull/1588/commits/45dbe709532851ac8cac53113bd1a6b258f0ad29